### PR TITLE
Add SOP for sync-rover-group

### PR DESCRIPTION
**Pull request by hongkailiu**
Follow up https://coreos.slack.com/archives/CHY2E1BL4/p1655733326558579

/cc @openshift/test-platform

### CODE REVIEW

File: docs/dptp-triage-sop/sync-rover-group.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+| [test-platform-ci-monitoring-viewers](https://rover.redhat.com/groups/group/test-platform-ci-monitoring-viewers) | ci-monitoring-viewers   | viewers of openshift-monitoring and prow-monitoring on all CI clusters |
+
+In theory, no groups except `test-platform-ci-admins` should be owned by the TP team.
+In reality, the groups `ci-sudoers` and `ci-monitoring-viewers` are there on the clusters before the support of Rover groups. The members are from different OCP teams.
+The Rover groups `test-platform-ci-sudoers` and `test-platform-ci-monitoring-viewers` are created to keep the permissions for the members.
+Until they are replaced by the groups they belong to, we have to maintain these two groups.
+
+Note that the group name `ci-monitoring-viewers` is misleading because they can modify alerts which is because it is bound to [monitoring-alertmanager-edit](../../clusters/build-clusters/common/monitoring-alertmanager-edit.yaml).
+There is [no read-only role to view alert-manager](https://coreos.slack.com/archives/C0VMT03S5/p1650628522595379?thread_ts=1650571924.565169&cid=C0VMT03S5).
```
Comment (hongkailiu) [inline docs/dptp-triage-sop/sync-rover-group.md:31]: Link to the RFE card?

Comment (hongkailiu) [reply docs/dptp-triage-sop/sync-rover-group.md]: Updated.

File: docs/dptp-triage-sop/sync-rover-group.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Sync Rover Groups
+
+## How it works
+
+Rover groups are synced to the clusters by two jobs:
+
+- A [cronjob/sync-rover-groups-update](https://console-openshift-console.apps.ocp-c1.prod.psi.redhat.com/k8s/ns/ocp-test-platform/batch~v1~CronJob/sync-rover-groups-update/) on the psi cluster: It stores the groups from Rover to `ConfigMap/sync-rover-groups` in `namespace/ci` of `app.ci`.
+
+-  A periodic `[ProwJob/periodic-github-ldap-user-group-creator](https://github.com/openshift/release/blob/master/ci-operator/jobs/infra-periodics.yaml)`: It maintains the groups defined in `ConfigMap/sync-rover-groups` on the clusters.
```
Comment (wking) [inline docs/dptp-triage-sop/sync-rover-group.md:9]: nit: you want the backticks inside the square brackets:

```markdown
[`ProwJob/periodic-github-ldap-user-group-creator`](https://github.com/openshift/release/blob/master/ci-operator/jobs/infra-periodics.yaml)
```

to get it rendered as a clickable link.

Comment (hongkailiu) [reply docs/dptp-triage-sop/sync-rover-group.md]: Updated.

**Pull request merged**